example/test_problem/Hydro/ClusterMerger/Input__TestProb

```diff
@@ -1,32 +1,68 @@
 # problem-specific runtime parameters
 Merger_Coll_NumHalos    2 # number of halos
+AGN_feedback            0 # turn on/off (1/0) AGN feedback
 
 # parameters for cluster 1
 Merger_File_Prof1       profile1_gamer.h5    # profile table of cluster 1
 Merger_File_Par1        1to3_b0.0_gamerp_1.h5    # particle file of cluster 1
+Unit_R1                 1.0 # the unit of length in cluster 1's IC file (in cgs)           
```

Comment:
I need to look at the rest of the code, but these `Unit_R*`, etc. seem a bit superfluous to me. I am not sure what version of `cluster_generator` you are using, but in the one on GitHub we have a function `setup_gamer_ics` that _always_ uses CGS. So unless there is a compelling reason to have parameters to say otherwise I would say we don't need these.